src/picongpu/submit/pizdaint-cscs/SLURM_Tutorial.md

```diff
@@ -0,0 +1,27 @@
+SLURM examples
```

Comment:
hm, the interactive part might be useful. you can just define

```bash
alias getInteractive="salloc --time=0:10:00 --nodes=1 --ntasks-per-node=16 --ntasks-per-core=2 --partition normal --ntasks-per-core=2"
```

in the `picongpu.profile.example` - I have this alias in very often for exactly that case.